openmeter/ent/schema/idprefix.go

```diff
@@ -0,0 +1,34 @@
+package schema
+
+// Table of ID prefixes for each entity type.
+// Please make sure you are picking a unique prefix for each entity type.
+// Recommendation:
+// - at least 3 characters of the entity type name.
+// - max length is 8 characters (field size limit)
```

Comment:
I suppose this is the hard part to get right. I'm not 100% scoping is the way to go, I'd rather define what to do in case of name collisions and use as short names as possible... wdyt?

Reply:
Yes, scoping is 100% TBD. Initially I was doing stuff like 
- BillingInvoiveItem => bii_
- Customer => c_

But then I didn't like the one-character names, so that's why I ended up with this scoping thing. I am not against dropping that part.

I would just mandate that there are no collisions, as seven characters should be enough.